### PR TITLE
#trivial Add kibana metric events to splunk as well

### PR DESCRIPTION
**Pull request by omninando**
#### What is the purpose of this pull request?
* Add splunk pageview events
* Add accountName to some events

#### What problem is this solving?
We are moving kibana events to splunk

#### How should this be manually tested?
n/a

#### Screenshots or example usage
n/a

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/actions/metric-actions.js [hunk 1]
```diff
@@ -169,6 +172,17 @@ export const searchedPickupAddressEvent = () => dispatch => {
       env: isEnv(VTEXBETA) ? VTEXBETA : VTEXSTABLE,
     },
   })
+
+  logSplunk({
+    level: IMPORTANT,
+    type: INFO,
+    workflowType: 'shipping',
+    workflowInstance: 'searchedPickupAddress',
```
Comment (kevinch) [inline react/actions/metric-actions.js]: Would some `const`s be useful for those `workflowInstance`s and/or `workflowType`s?

Comment (omninando) [reply react/actions/metric-actions.js]: Agreed

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Splunk events to metrics which were logged in kibana
```
Comment (kevinch) [inline CHANGELOG.md]: Nitpick
```suggestion
- Splunk events to metrics which were logged in Kibana
```

File: react/actions/metric-actions.js [hunk 3]
```diff
@@ -29,6 +30,37 @@ evidenceClient.config({
...
+    splunkEvents.logEvent(
+      level,
+      type,
+      workflowType,
+      workflowInstance,
+      event,
+      account
+    )
+  } else {
+    console.log('Log', config)
```
Comment (kevinch) [inline react/actions/metric-actions.js]: Normal?

Comment (omninando) [reply react/actions/metric-actions.js]: yes

**Pull request merged**